cd /home/mkoeppe/w/latte/build-gcc411-64/lidia-2.2.0/src/linear_algebra/include/LiDIA/matrix/
diff -u /home/mkoeppe/w/latte/distro/lidia-2.2.0/src/linear_algebra/include/LiDIA/matrix/dense_bigint_matrix_modules.cc /home/mkoeppe/w/latte/build-gcc411-64/lidia-2.2.0/src/linear_algebra/include/LiDIA/matrix/dense_bigint_matrix_modules.cc
--- src/linear_algebra/include/LiDIA/matrix/dense_bigint_matrix_modules.cc	2006-03-06 13:24:12.000000000 +0100
+++ src/linear_algebra/include/LiDIA/matrix/dense_bigint_matrix_modules.cc	2006-07-21 13:24:10.614757000 +0200
@@ -2472,7 +2472,7 @@
 column_oriented_dense_matrix_modules< T >::sorting_gcd_plus_minimal_norm (const MR< T > &A,
 									  lidia_size_t startr,
 									  lidia_size_t startc,
-									  lidia_size_t &pos,
+									  lidia_size_t &index,
 									  lidia_size_t norm) const
 {
 	lidia_size_t FOUND = 0;

Diff finished at Fri Jul 21 13:26:26
